projects/angular-simple-oidc/src/lib/oidc-code-flow-client.service.ts

```diff
@@ -79,6 +87,7 @@ export class OidcCodeFlowClient {
                     clientId: config.clientId,
                     scope: config.scope,
                     responseType: 'code',
+                    acrValues: config.acrValues ?? '',
                     ...params,
                 })),
```

Comment:
chore: improve the way we set the acrValues to avoid chaging tests

const codeFlowMetadata =  {
                    clientId: config.clientId,
                    scope: config.scope,
                    responseType: 'code', 
                    ...params,
}

if(config.acrValues) {
       codeFlowMetadata.acrValues = config.acrValues
}

return codeFlowMetadatañ

Reply:
Done!